src/OrchardCore/OrchardCore.DisplayManagement/Extensions/IHtmlContentExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Html;
+
+
+namespace OrchardCore.DisplayManagement.Extensions;
+
+public static class IHtmlContentExtensions
```

Comment:
```suggestion
public static class HtmlContentExtensions
```

Reply:
Changed the name of the class and the name of the file.